pkg/tenant/projects.go

```diff
@@ -156,6 +156,10 @@ func (s *Server) TenantProjectCreate(c *gin.Context) {
 		Name:     project.Name,
 	}
 
+	if project.Description != "" {
```

Comment:
The Figma design doesn't include a max word count, but I'm wondering if we should add a check for that.

Reply:
Never mind. In SC-15723, the max length for the description is listed as 2k characters.